.github/CODEOWNERS

```diff
@@ -223,6 +223,7 @@
 /devenv/local-npm/ @grafana/frontend-ops
 /devenv/vscode/ @grafana/frontend-ops
 /devenv/setup.sh @grafana/grafana-backend-services-squad
+devenv/plugins.yaml @grafana/plugins-platform-frontend
```

Comment:
Just double-checking here - I guess we don't need the leading slash in front of `devenv/plugins.yaml` right?

Reply:
oh. looks like the build passed, so the file was picked up. but I'm changing this. thanks for finding this!